src/ansible/playbook_image_service.yml

```diff
@@ -30,6 +30,7 @@
   - { role: passkey, when: ansible_distribution == "Ubuntu" or ansible_distribution == "Debian" }
   - { role: virtsmartcard, when: ansible_distribution != "Ubuntu" and ansible_distribution != "Debian" and virt_smartcard }
   - { role: ipasmartcard, when: ansible_os_family == 'RedHat' and virt_smartcard }
+  - { role: selenium, when: ansible_distribution != "Ubuntu" and ansible_distribution != "Debian" and selenium_support }
```

Comment:
I think the conditions can be simplified by `when: ansible_os_family == 'RedHat'`